.github/workflows/clean-rev-image.yml

```
@@ -0,0 +1,96 @@
+        
+        # Extract tags and count them
+        tag_count=$(echo $response | jq '.results | length')
+
+        # Set output based on the number of tags
+        echo "auth_service_tag_count=$tag_count" >> $GITHUB_ENV
+        echo "auth_service_tags=$response" >> $GITHUB_ENV
+
+    # Delete old tags for user-service if more than 3 tags exist
+    - name: Delete old tags for user-service if more than 3
+      if: env.user_service_tag_count > '3'
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:5:42: Double quote to prevent globbing and word splitting

(shellcheck)